mcp-server-quill/src/mcp_server_quill/dependencies.py

```
@@ -0,0 +1,24 @@
+from typing import Annotated
+
+from fastapi import Depends
+
+from mcp_server_quill.config import get_app_settings
+from mcp_server_quill.quill.client import QuillAPI
+from mcp_server_quill.quill.search import TokenSearchAPI
+
+
+async def get_quill_client() -> QuillAPI:
```

Ruff (0.14.14)

[warning] Missing docstring in public function

(D103)